### PR TITLE
Branch meet

### PR DESCRIPTION
**Pull request by meejain**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #[<gh-issue-id>](https://github.com/mkclawson/impala/issues/2)

Test URL:

https://branch-meet--impala--mkclawson.hlx.page/Drafts/Meet/

Major tasks done - 

1) Created our own GIT Branch, impacting just the dummy index file under Drafts/Meet folder. 
2) Fixing the position & size of HERO image via adjusting few parameters in CSS including the background colour. 
3) Worked on NAV (Header) & adjusted its width, also fixed flex - justify content value in order to meet the source site. 
4) Further broke NAV bar into separate sections, move one of them from block to flex mode and then adjust the rest of the dependent parameters.

### CODE REVIEW

File: blocks/header/header.css [hunk 1]
```diff
@@ -15,7 +15,7 @@ header nav {
   align-items: center;
   gap: 0 2em;
   margin: auto;
-  max-width: 1264px;
+  max-width: 1700px;
```
[the remark this answers is not in the document]
Comment (meejain) [reply blocks/header/header.css:18]: Yeah the flex segment of the header was extending far away from the sides of the HERO image and hence I need to put a value max-width - 1700 px in order to limit that. Please advise.

File: styles/styles.css [hunk 2]
```diff
@@ -14,13 +14,13 @@
...
-  --background-color: #fff;
+  --background-color: #000000;
   --overlay-background-color: #eee;
   --highlight-background-color: #ccc;
-  --text-color: #000;
+  --text-color: white;
 
   /* fonts */
-  --body-font-family: 'helvetica neue', helvetica, ubuntu, roboto, noto, sans-serif;
+  --body-font-family: 'Playfair display', Didot, Raleway, roboto, noto, sans-serif;
```
[the remark this answers is not in the document]
Comment (meejain) [reply styles/styles.css]: Sure. Revert back the changes for now.

File: styles/styles.css [hunk 3]
```diff
@@ -241,3 +243,24 @@ main .section {
...
+
+#coming-back-summer-of-2022{
+font-size: 70px;
+}
+
+main > .section > .default-content-wrapper > h1 {
+  font-weight: normal;
+}
+
+#you-want-your-own-supernatural-impala-but-cant-afford-100000-to-buy-one--cant-find-a-1967-chevrolet-impala-even-if-you-could-afford-one--come-share-in-my-love-for-baby-and-take-advantage-of-the-years-of-work-put-into-this-beautiful-car--have-a-private-photo-shoot-be-driven-to-a-wedding-cosplay-party-prom-or-other-special-event--be-the-envy-of-all-your-supernatural-friends-and-create-a-special-event-special-memories-that-will-last-forever, #email-babysupernaturalimpalacom, #text-385-214-5306-and-talk-to-mike-for-more-info, #contact-us{
```
Comment (helms-charity) [inline styles/styles.css:255]: What the hell is this?

File: styles/styles.css [hunk 4]
```diff
@@ -241,3 +243,24 @@ main .section {
 main .section.highlight {
   background-color: var(--highlight-background-color);
 }
+
+#coming-back-summer-of-2022{
```
Comment (helms-charity) [inline styles/styles.css:247]: no, too specific. You need to target an element that doesn't change when the text changes.

File: styles/styles.css (hunk 3 above)
Comment (meejain) [reply styles/styles.css]: Yes , I was just testing that. I did change them with proper CSS notations, though - 

main > .section > .default-content-wrapper > h1
main > .section > .default-content-wrapper > h4
main > .section > .default-content-wrapper > h3

File: styles/styles.css (hunk 4 above)
Comment (meejain) [reply styles/styles.css]: @helms-charity  - Yes, fixed that via 

main > .section > .default-content-wrapper > h1:first-of-type

**Comment by meejain** [comment]
Working on the issues - 

https://github.com/mkclawson/impala/issues/2
https://github.com/mkclawson/impala/issues/3

**Comment by meejain** [comment]
https://github.com/mkclawson/impala/issues/2  & https://github.com/mkclawson/impala/issues/5 are taken care by this pull request.

File: styles/styles.css [hunk 5]
```diff
@@ -14,13 +14,15 @@
...
+  --overlay-background-color: #000;
   --highlight-background-color: #ccc;
-  --text-color: #000;
+  --text-color: white;
 
   /* fonts */
-  --body-font-family: 'helvetica neue', helvetica, ubuntu, roboto, noto, sans-serif;
+
+  /* --body-font-family: 'helvetica neue', helvetica, ubuntu, roboto, noto, sans-serif; */
+  --body-font-family: 'Playfair display', 'Didot', 'Raleway', roboto, noto, sans-serif;
```
Comment (helms-charity) [inline styles/styles.css:25]: If you can't import Playfair Display, can you replace it with another font that is free and looks close/similar?

File: styles/styles.css [hunk 6]
```diff
@@ -58,6 +60,7 @@ body {
   margin: 0;
   font-family: var(--body-font-family);
   line-height: 1.6;
+  text-align: center;
```
Comment (helms-charity) [inline styles/styles.css:63]: Making the entire site's body text centered might not be desirable, but we can leave it.

File: styles/styles.css [hunk 7]
```diff
@@ -241,3 +243,24 @@ main .section {
...
+}
+
+main > .section > .default-content-wrapper > h1 {
+  font-weight: normal;
+}
+
+main > .section > .default-content-wrapper > h4 {
+  font-weight: normal;
+  font-size: 15px;
+  font: normal normal normal 14px/1.79em raleway,sans-serif;
```
[the remark this answers is not in the document]
Comment (meejain) [reply styles/styles.css]: The above line was removed yesterday. The new code is - 

main > .section > .default-content-wrapper > h4 {
  font-weight: normal;
  font-size: 15px;
  font-family: Raleway, sans-serif;
  letter-spacing: 2px;
  line-height: 1.6;
  }

File: styles/styles.css (hunk 5 above)
Comment (meejain) [reply styles/styles.css:25]: Imported and committed

**Pull request merged**